### PR TITLE
F/8 display current song artist

### PR DESCRIPTION
**Pull request by lauraturk**
### What does it do?
Adds a 'what' command to display currently playing song to whole channel
Current song also displays when 'play' command is used
added prettier

### Any helpful background information?
popular vote seems to want new songs added at index 0. Removed the invoking of play from queue that interrupts currently playing song, but newly added song should play next.

### New dependencies? What are they used for?
prettier for prettying

### Definition of Done
two use cases for issue were resolved: showing currently playing w/ play command and newly added what command

Could not test formatting of currently playing song in slack channel, may need some tweaking once pushed to production.

### What are the relevant tickets?
#8 

### Who helped?
@budaminof

### CODE REVIEW

File: controllers/spotifyController.js [hunk 1]
```diff
@@ -121,122 +132,166 @@ const SpotifyController = (function() {
...
-        'Authorization': 'Bearer ' + oauthAccessToken
+        Authorization: 'Bearer ' + oauthAccessToken,
       },
-      json: true
+      json: true,
+      resolveWithFullResponse: true,
     };
 
-    if(playlistId) {
+    /*if(playlistId) {
```
Comment (chicagoing) [inline controllers/spotifyController.js]: Remind me, why don't we need this anymore?

File: controllers/spotifyController.js [hunk 2]
```diff
@@ -308,20 +373,25 @@ const SpotifyController = (function() {
...
       const fn = () => {
         switch (text) {
           case 'pause':
             pause(res);
             break;
           case 'play':
-            play();
-            res.send('play resumed!');
+            play(res);
+            //res.send('play resumed!');
```
Comment (chicagoing) [inline controllers/spotifyController.js]: You can remove this line if it's not going to be used.

File: package.json [hunk 3]
```diff
@@ -23,7 +23,10 @@
     "url": "git+https://github.com/chicagoing/accelerator-playlist.git"
   },
   "scripts": {
-    "start": "nodemon ./bin/www"
+    "start": "nodemon ./bin/www",
+    "prettier": "prettier --single-quote --trailing-comma=es5 --list-different --write es5 './**/*.js'"
   },
-  "devDependencies": {}
+  "devDependencies": {
+    "prettier": "1.7.4"
```
Comment (chicagoing) [inline package.json]: Awesome, thank you!

File: controllers/spotifyController.js (hunk 1 above)
Comment (lauraturk) [reply controllers/spotifyController.js]: play was only being called w/ playlistId as an argument from queue which was causing the current song to be interrupted by the newly added song. We left it commented until we figure out how best to manage adding songs to the playlist.

**Pull request merged**